app/components/directives/address.js

```diff
@@ -0,0 +1,14 @@
+(
+    function(){
+        angular.module('MockApp')
+            .component('address',{
+                template:
+                    '<b>Street</b> : {{$ctrl.model.street}}<br/> '+
+                    '<b>City</b>: {{$ctrl.model.city}}<br/>'+
+                    '<b>Zipcode</b>: {{$ctrl.model.zipcode}} ',
```

Comment:
Create a separate html file for this template and inject the template location here